examples/CMakeLists.txt

```diff
@@ -105,7 +105,7 @@ enable_testing()
 ### primitive tests with bips98_606 graph
 add_test(NAME TEST_ASTAR COMMAND astar market
   ${gunrock_INCLUDE_DIRS}/dataset/small/map/cities.mtx
-  --mapfile=${gunrock_INCLUDE_DIRS}/dataset/small/map/cities.map)
+  --mapfile=${gunrock_INCLUDE_DIRS}/dataset/small/map/cities.map --undirected)
```

Comment:
Look at why directed is failing:
https://github.com/gunrock/gunrock/blob/dev/examples/astar/test_astar.cu#L615-L623